.github/workflows/windows.yml

```diff
@@ -319,7 +319,7 @@ jobs:
     defaults:
       run:
         shell: pwsh
-    runs-on: windows-2022
+    runs-on: aks-win-4-cores-8gb-staging
```

Comment:
is it OK to commit staging runners to master branch?

Reply:
we are depends on -https://github.com/openvinotoolkit/openvino/pull/29215, but we can use staging runners to stabilize our CI  and will switch it to the common runners later, when they are ready.